tokenserver/tests/test_service.py

```diff
@@ -190,10 +191,20 @@ def test_discovery(self):
         })
 
     def test_version_returns_404_by_default(self):
+        # clear cache
+        try:
+            del tokenserver.views.version_view.__json__
+        except AttributeError:
+            pass
```

Comment:
These tests started failing now that we run them twice, thanks to caching. Bleh.